pkg/devicemapper/devmapper_wrapper.go

```diff
@@ -188,8 +188,11 @@ func dmTaskGetDepsFct(task *CDmTask) *Deps {
 		Count:  uint32(Cdeps.count),
 		Filler: uint32(Cdeps.filler),
 	}
-	for _, device := range Cdeps.device {
-		deps.Device = append(deps.Device, (uint64)(device))
+	// this gem from rsc
+	// https://groups.google.com/d/msg/golang-dev/iybwRUoA1Vg/mAvIrl9rBwAJ
+	devices := (*[1 << 30]uint64)(unsafe.Pointer(uintptr(unsafe.Pointer(Cdeps)) + unsafe.Sizeof(*Cdeps)))[:Cdeps.count]
```

Comment:
As much as I trust rsc, I have no idea what `1<<30` refers to and it's really hard to understand why this makes it a slice. I'd still prefer reflect.SliceHeader, it's much more readable.